apps/managedidentity/managedidentity.go

```diff
@@ -64,13 +71,20 @@ func (c UserAssignedClientID) value() string   { return string(c) }
 func (o UserAssignedObjectID) value() string   { return string(o) }
 func (r UserAssignedResourceID) value() string { return string(r) }
 func SystemAssigned() ID {
-	return systemAssignedValue("")
+	return systemAssignedValue(systemAssignedManagedIdentity)
 }
 
 type Client struct {
 	httpClient ops.HTTPClient
 	miType     ID
 	// source     Source reenable when required in future sources
+	cacheManager manager
+}
+
+type manager interface {
```

Comment:
Do we still need this? We discussed about not offering serialziation API for managed identity.

Reply:
This is private interface, not exposed for outside. We need this so that we can have access to cache storage in client.

Reply:
Removed.